### PR TITLE
NCA inflection

### PR DESCRIPTION
**Pull request by erluti**
## Description of change
Facilities are named with acronyms, `NCA` didn't have the corresponding inflection.  This will ensure the `app/models/facilities/nca_facility.rb` file will load with zeitwerk enabled.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#10976
department-of-veterans-affairs/va.gov-team#10165

## other notes
* fixed ordering a duplicate inflections

### CODE REVIEW

File: config/initializers/inflections.rb [hunk 1]
```diff
@@ -15,6 +14,7 @@
   inflect.acronym 'GI'
   inflect.acronym 'IHub'
   inflect.acronym 'MDOT'
+  inflect.acronym 'NCA' # National Cemetery Administration
```
Comment (johnpaulashenfelter) [inline config/initializers/inflections.rb:17]: This literally doesn't need any filename/locations changed? Or a 'require' removed?

Comment (erluti) [reply config/initializers/inflections.rb:17]: no it only occurs in the `app/models/facilities/nca_facility` file, and it won't load when zeitwerk is turned on unless we rename the class to `NcaFacility`

**Comment by omgitsbillryan** [comment]
Bump - any reason not to merge?

**Comment by erluti** [comment]
I was holding off for more camelized-schema coverage, but I'm confident this won't touch anything.  I'll add a camel-inflected schema test for `spec/request/v0/facilities/va_request_spec.rb` next and then I'll merge this one.  Mostly just nervous to merge on a Friday after we rolled back all these changes.

**Comment by erluti** [comment]
Spec that excercises a camelized version of facilities endpoint https://github.com/department-of-veterans-affairs/vets-api/pull/4677

**Pull request merged**